.github/workflows/ci.yml

```diff
@@ -260,6 +260,28 @@ jobs:
         name: Install nvim
         run: ./ci/run_tests.sh install_nvim
 
+      - if: matrix.flavor != 'tsan' && matrix.flavor != 'functionaltest-lua' && !cancelled()
+        name: Generate test-runners
+        run: |
+          cmake --build $GITHUB_WORKSPACE/build --target generate-test-runners
+
+
+      # macOS runners have 3 threads
```

Comment:
Why are we fiddling with threads? What evidence is there that the test runner is CPU-bound (not IO-bound)? How much faster is the test run with parallel invocation?

Reply:
on the CI, this is a bit harder to gauge, but here are the stats on my machine

<table>
<tr>
	<td>cmd
	<td>time
<tr>
	<td>TEST_FILE=test/functional/api cmake --build build --target functionaltest
	<td> 11s
<tr>
	<td>ctest --test-dir build -L 'spec' --repeat until-pass:3 -j8 --progress
	<td>8.75s
<tr>
	<td>ctest --test-dir build -L 'spec' --repeat until-pass:3 -j16 --progress
	<td>5.85s
</table>